### DISCUSSION TITLE
Image Limit?

### DISCUSSION THREAD

**Discussion by netdzynr** [zanllp/infinite-image-browsing#600] [Q&A]
Am running from a Python installation in a browser on MacOS and am wondering if there's a practical limit to the number of images that are capable of being indexed. I run up against a max of 3058 thumbnails with multiple browser refreshes.

BTW, I'm using this implementation because I didn't see a Mac version provided among the standalone downloads. Please let me know if I'm missing it. Thank you.

**Comment by zanllp** [comment]
There's no limit. On my computer, I generated an index for over 34,000 images. I see that for some users, their iib.db has reached up to 1GB, which should correspond to an index of approximately 500,000 images. If your images are not being indexed, please make sure you have added the folder containing the images to the search scan path.

**Comment by zanllp** [comment]
The reason there is no Mac version is due to unresolved permission issues from before, and currently, I don't have a Mac device, making it difficult to debug.
